### ISSUE TITLE
alt field for hero images

### ISSUE THREAD

**Issue by ctot-nondef** [acdh-oeaw/website-clarink#1]
### Is your feature request related to a problem? Please describe.

the hero images currently lack the possibility to add an alt text

### Describe the solution you'd like

add a field to the content model
